history.txt

```diff
@@ -1,6 +1,21 @@
 OMERO version history
 =====================
 
+5.1.0-m5 (March 2015)
+---------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.1.0. Use at your own risk**.
+
+Changes include:
+
+-  implementation of OMERO.mail for emailing users via the server
```

Comment:
Is there anything I should do in the docs PR to make this linkable? or is the formatting for that just too much for the history page?